src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -944,7 +941,7 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 			if (replaceEditorArg.editor instanceof EditorInput) {
 				const replacementArg = replaceEditorArg as IEditorReplacement;
 				if (replacementArg.options?.override !== EditorOverride.DISABLED && targetGroup) {
-					const override = await this.editorOverrideService.resolveEditorOverride(replacementArg.replacement, replacementArg.options, targetGroup);
+					const override = await this.editorOverrideService.resolveEditorOverride({ resource: replacementArg.replacement.resource, options: replacementArg.options }, targetGroup);
```

Comment:
This is dangerous, no? Because we have a `EditorInput` in hand but you only take it's `resource` going forward. What about `SideBySideEditorInput` and `DiffEditorInput`? Because in the `resolveEditorOverride` method we seem to be supporting side by side?

https://github.com/microsoft/vscode/blob/48cbb9430a7d6089d5e6ddb52cc5e1213b317cba/src/vs/workbench/services/editor/browser/editorOverrideService.ts#L83-L83

Reply:
I think this no longer applies